api-compact.yaml

```diff
@@ -191,18 +188,6 @@ auth_header: &auth_header
   required: true
   type: "string"
 
-limit_qs_parameter: &limit_qs_parameter
```

Comment:
are we removing these query params because they only apply to the catalog endpoints? seems like they could apply to any endpoint... and in fact these are the preferred query params to use for pagination.